notifiers/providers/email.py

```diff
@@ -143,7 +158,9 @@ def _send_notification(self, data: dict) -> Response:
             if not self.configuration or not self.smtp_server or self.configuration != configuration:
                 self._connect_to_server(data)
             email = self._build_email(data)
-            self.smtp_server.sendmail(data['from'], data['to'], email.as_string())
+            if data['attachments']:
```

Comment:
If you remove the default then don't forget to change this to `data.get('attachments')`